RockWeb/Webhooks/Mailgun.ashx

```diff
@@ -78,19 +78,26 @@ public class Mailgun : IHttpHandler
             
             if ( !string.IsNullOrWhiteSpace( request.Form["workflow_action_guid"]))
             {
-                Guid? actionGuid = request.Form["workflow_action_guid"].AsGuidOrNull();
+                Guid? actionGuid = request.Form["workflow_action_guid"].Split( ',' )[0].AsGuidOrNull();
                 string status = string.Empty;
                 switch ( eventType )
                 {
-                    case "complained":
-                    case "unsubscribed":
+                    case "complained": break;
+                    case "unsubscribed": break;
                     case "delivered": status = SendEmailWithEvents.SENT_STATUS; break;
```

Comment:
My mistake, those breaks for "complained", "unsubscribed" and "dropped" should not be there as the delivered case should catch the first two and bounced should catch dropped.